src/platform_impl/macos/appkit/event.rs

```diff
@@ -123,13 +152,34 @@ extern_methods!(
         #[sel(stage)]
         pub fn stage(&self) -> NSInteger;
 
+        #[sel(isARepeat)]
+        pub fn is_a_repeat(&self) -> bool;
+
+        #[sel(windowNumber)]
+        pub fn window_number(&self) -> NSInteger;
+
+        #[sel(timestamp)]
+        pub fn timestamp(&self) -> NSTimeInterval;
+
         pub fn characters(&self) -> Option<Id<NSString, Shared>> {
             unsafe { msg_send_id![self, characters] }
         }
 
         pub fn charactersIgnoringModifiers(&self) -> Option<Id<NSString, Shared>> {
             unsafe { msg_send_id![self, charactersIgnoringModifiers] }
         }
+
+        pub fn lalt_pressed(&self) -> bool {
+            const NX_DEVICELALTKEYMASK: u32 = 0x00000020;
```

Comment:
Nit: This would be very nice to have documented, including a link to the [source](https://github.com/apple-oss-distributions/IOHIDFamily/blob/IOHIDFamily-1892.60.4/IOHIDSystem/IOKit/hidsystem/IOLLEvent.h#L258-L259).

And e.g. put it next to `NSEventModifierFlags`.

Reply:
You'd be surprised how many of the sources are there for it. It's basically from `CGEventFlags` from what I know or similar. It least this part of the code is also present in the `minifb`.

Reply:
I've added just so it could be found in case someone decides to expand the functionality.